src/Builder.php

```diff
@@ -72,6 +75,7 @@ public function __construct(
         $this->buildDir = $buildDir;
         $this->filesToCopy = $filesToCopy;
         $this->files = new Filesystem();
+        $this->mimeTypes = new MimeTypes();
```

Comment:
Perhaps inject `MimeTypesInterface` instead, so anyone can override it globally if required / rely on default one registered in their app.